.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -988,4 +1006,12 @@ sqs_Scenario_TopicsAndQueues:
                 - cpp.example_code.cross-service.topics_and_queues.messaging_with_topics_and_queues
   services:
     sns: {CreateTopic, Subscribe, Publish, Unsubscribe, DeleteTopic}
-    sqs: {CreateQueue, GetQueueAttributes, SetQueueAttributes, ReceiveMessage, DeleteMessageBatch, DeleteQueue}
+    sqs:

```

Comment:
Question only - does this format with the line breaks seem inconsistent for yaml?

Reply:
My auto format did it. It might be inconsistent but it's easier to read IMO and still works, so probably not an issue?